.github/workflows/ci.yml

```diff
@@ -27,6 +27,7 @@ jobs:
         run: |
           sudo cpanm --notest Test::Nginx > build.log 2>&1 || (cat build.log && exit 1)
           sudo luarocks install luacheck
+          sudo luarocks install api7-lua-resty-http 0.2.2-0
```

Comment:
we'd better add `xrpc`, it is meaningful

Reply:
The `resty.http` is only used in testing.
After merging the PR here, the healthcheck library will be released and the xrpc suffix will be added.

Reply:
![Uploading image.png…]()